signer/src/transaction_coordinator.rs

```diff
@@ -1028,8 +1035,35 @@ mod tests {
     #[tokio::test]
     async fn should_be_able_to_coordinate_signing_rounds() {
         test_environment()
-            .assert_should_be_able_to_coordinate_signing_rounds()
+            .assert_should_be_able_to_coordinate_signing_rounds(tokio::time::Duration::ZERO)
+            .await;
+    }
+
+    #[tokio::test]
+    async fn should_wait_before_processing_bitcoin_blocks() {
+        // NOTE: Above test `should_be_able_to_coordinate_signing_rounds`
+        // could be removed as redundant now.
+
+        // Measure baseline.
+        let baseline_start = std::time::Instant::now();
+        test_environment()
+            .assert_should_be_able_to_coordinate_signing_rounds(tokio::time::Duration::ZERO)
+            .await;
+        // Locally this takes a couple seconds to execute.
+        // This truncates the decimals.
+        let baseline_elapsed = tokio::time::Duration::from_secs(baseline_start.elapsed().as_secs());
+
+        let delay_i = 3;
+        let delay = tokio::time::Duration::from_secs(delay_i);
+        std::env::set_var(
+            "SIGNER_SIGNER__BITCOIN_PROCESSING_DELAY",
+            delay_i.to_string(),
+        );
```

Comment:
Not sure if there's a better way to do this.

Reply:
I think I'd make the delay config explicitly in seconds as a uint.

Reply:
That's what chatgpt is for :rofl: